src/client/Microsoft.Identity.Client/ManagedIdentity/ManagedIdentitySource.cs

```diff
@@ -54,20 +72,20 @@ public virtual async Task<ManagedIdentityResponse> AuthenticateAsync(AppTokenPro
                     return GetSuccessfulResponse(response);
```

Comment:
seems like we are ignoring clientID in this response and not passing it on to MSALTokenResponse. MSI Endpoint gives clientID as well. Do we know if this needs to be passed on?

Reply:
We can only send `AccessToken`, `ExpiresIn` and `RefreshIn` to MSAL through [AppTokenProviderResult.cs](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/src/client/Microsoft.Identity.Client/Extensibility/AppTokenProviderResult.cs#L13)